lib/Backends/Interpreter/Interpreter.h

```diff
@@ -41,7 +41,7 @@ class Variable;
 /// execute the instructions one at a time.
 class Interpreter final : public Backend {
   /// The Module that holds the IR. This does not own the module.
-  const IRFunction *F_;
+  std::unique_ptr<const IRFunction> F_;
```

Comment:
Is that really a "Module"? Seems that module contains list of Functions* but not IRFunction.

Reply:
Looks like a wrong comment got copy-pasted everywhere.